packages/tests/src/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function areUint8ArraysEqual(a: Uint8Array, b: Uint8Array): boolean {
```

Comment:
nit: maybe some package can handle that for us as well as other utils we have specific for tests?

Reply:
Thanks for the suggestion, I found [lodash](https://github.com/lodash/lodash) for this specific function and started using it
Will review other "generic" custom test utils if they can be replaced with known packages but in another PR because this one is already big enough